medusa/network_timezones.py

```diff
@@ -104,11 +107,15 @@ def load_network_dict():
     """Load network timezones from db into dict network_dict (global dict)."""
     try:
         cache_db_con = db.DBConnection('cache.db')
-        cur_network_list = cache_db_con.select('SELECT * FROM network_timezones;')
-        if not cur_network_list:
+        sql_result = cache_db_con.select('SELECT network_name, timezone FROM network_timezones;')
+        if not sql_result:
             update_network_dict()
-            cur_network_list = cache_db_con.select('SELECT * FROM network_timezones;')
-        d = dict(cur_network_list)
+            sql_result = cache_db_con.select('SELECT network_name, timezone FROM network_timezones;')
+
+        d = {}
+        for row in sql_result:
+            d[row['network_name']] = row['timezone']
```

Comment:
Same as above (instead of lines 115-117):
```py
    d = {
        row['network_name']: row['timezone']
        for row in sql_result
    }
```

Reply:
changed